include/menu.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <vector>
+#include "game.h"
+
+class Menu {
+   private:
+    size_t numberOfOptions{3};
+    size_t selectedOptionIndex{0};
```

Comment:
`size_t selected_action_index = 0`

Jeśli chcemy używać _uniform initialization syntax_, to używajmy jej wszędzie, albo nigdzie. Na razie pójdźmy drogą "(prawie)nigdzie", a potem najwyzej w jakimś 1 PRku sie wszystko zmieni na _uniform init_